tools/build_travis.py

```diff
@@ -260,6 +260,12 @@
                "usb"  : ["USB_1", "USB_2" ,"USB_3"],
                }
      },
+    {"target": "DISCO_L072CZ_LRWAN1",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_16"],
+               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
```

Comment:
Note: as preparation for upcoming RTX update, these rtos tests from these scripts will be removed (tests will be in the codebase but executed via mbed cli).